docs-source/content/userguide/managing-domains/model-in-image/model-files.md

```diff
@@ -83,6 +85,9 @@ For a description of model file macro references to secrets and environment vari
       AdminUserName: '@@SECRET:__weblogic-credentials__:username@@'
       AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'
     ```
+
+- A JRF domain type model __must__ contain a `domainInfo.RCUDbInfo` stanza, see [Requirements for JRF domain types]({{< relref "/userguide/managing-domains/model-in-image/usage/_index.md#requirements-for-jrf-domain-types" >}}).
```

Comment:
stanza, see -> stanza; see (these are actually two sentences; if you want them joined, then use a semi-colon.)